src/resources/Pipelines/StatementGroups/StatementGroupsInterfaces.ts

```diff
@@ -44,7 +44,11 @@ interface StatementGroupModelBase {
      * The definition of the condition that must be satisfied for a request to be processed by the statement group.
      */
     conditionDefinition?: string;
-
+    /**
+     * A structured representation of the condition
+     */
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    conditionDetailed?: any;
```

Comment:
My only concern with this PR is that using `any` and switching in the future for a real type would probably be considered breaking. Do we know what the real type is? Could we use unknown instead?

Reply:
No I don't have the types yet. The API doesn't provide a schema for the detailed condition object. It is currently being worked on but I'm not sure when it will be completed. 🤷

Reply:
Replaced with `unknown` ✅